ivy_models_tests/bert/test_bert.py

```diff
@@ -0,0 +1,28 @@
+from transformers import AutoTokenizer, AutoModel
+import ivy
+import pytest
+import numpy as np
+from ivy_models.bert import bert_base_uncased
+
+
+@pytest.mark.parametrize("batch_shape", [[1]])
+@pytest.mark.parametrize("load_weights", [False, True])
+def test_bert(device, f, fw, batch_shape, load_weights):
+    """Test Bert Base Sequence Classification"""
+
+    num_dims = 768
+    tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+
+    inputs = tokenizer("hello, my cat is cute.", return_tensor="pt")
+    inputs = (
+        {k: v.to(device) for k, v in inputs.items()} if device is not None else inputs
+    )
+    ref_model = AutoModel.from_pretrained("bert-base-uncased")
```

Comment:
we try to avoid extra dependencies in the tests, do you think we are able to get some simple hardcoded native results here instead of instantiate the model itself?

Reply:
I can create some results and save them in the disk instead of instantiating the model